### PR TITLE
move Genesis to a node module

### PR DESCRIPTION
**Pull request by TobiTenno**
The code move is there, but there's some stuff that has to be done as a publisher.

That information can be found [here](https://docs.npmjs.com/getting-started/publishing-npm-packages).

For now, this can be tested by pulling, and running `npm link <path to cloned repo>` and `npm install` in the cloned repo.

At present, no CI is set up, although I added a sample link for it to README.md, but I'll be adding an issue for that as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,69 @@
 #Cephalon Genesis
-a Warframe telegram bot, built on the hubot framework
+A Warframe tracking bot, built on the hubot framework
 
-launch with `TELEGRAM_TOKEN=token TELEGRAM_WEBHOOK=https://example.com/hubot/telegram/receive MONGODB_URL=url bin/hubot -a telegram --name "somethingbot" -l /`
+For Telegram, launch with 
+
+`TELEGRAM_TOKEN=token TELEGRAM_WEBHOOK=https://example.com/hubot/telegram/receive MONGODB_URL=url bin/hubot -a telegram --name "somethingbot" -l /`
```
Comment (nspacestd) [inline README.md]: I'd leave out adapter-specific environment variables

File: README.md.save [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Cephalon Genesis
```
Comment (nspacestd) [inline README.md.save]: This file can be removed

Comment (TobiTenno) [inline README.md.save]: was planning on it, git didn't take my rm, it's fixed in the newest commit.

File: .gitignore.save [hunk 3]
```diff
@@ -0,0 +1,4 @@
+node_modules
```
Comment (nspacestd) [inline .gitignore.save]: This file can be removed too

**Comment by nspacestd** [comment]
I'd remove the /help command and let hubot-help handle that

**Pull request merged**